### PR TITLE
Create pending role

### PR DESCRIPTION
**Pull request by AnsonLee99**
### Administrative Info
Monday Board ID: 1181559666
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
1) Created pending role with no permissions
2) Newly created accounts will now default to pending role
3) Fixed account creation process by adding default values for the required graduation, github_username, and phone fields

### Testing
Created new accounts and checked that they defaulted to a pending role.

### Confirmation of Change 
Newly created account:
image

Pending role:
image

### CODE REVIEW

File: backend/services/users.js [hunk 1]
```diff
@@ -22,10 +22,20 @@ async function createUser(raw_user) {
   if (user) {
     throw ServiceError(409, "Email already taken");
   }
-  const raw_user_no_secret = { ...raw_user };
+
+  const pending_role = await Role.findOne({ name: "Pending" }).exec();
+  const raw_user_no_secret = {
+    ...raw_user,
+    phone: "(xxx)xxx-xxxx",
```
Comment (hahsu) [inline backend/services/users.js:29]: Can we add a TODO comment, to update this in the future?

**Review by hahsu — APPROVED** [review]
lgtm, other than adding that TODO comment. Thanks!

**Review by justinyaodu — APPROVED** [review]
LGTM. I think we'll have to make some changes to this setup eventually (similar to the changes for the [unassigned role](https://triton-software-engineering.monday.com/boards/1181559666/pulses/1297503259)), but those would probably be out of scope for this PR.

**Review by petabite — APPROVED** [review]
LGTM!

File: backend/services/users.js (hunk 1 above)
Comment (AnsonLee99) [reply backend/services/users.js:29]: done!

**Pull request merged**